.github/workflows/validatePR.yml

```
@@ -25,13 +25,20 @@ jobs:
           text: ${{ github.event.pull_request.body }}
           regex: "#[0-9]+"
           flags: gm
+      - uses: actions-ecosystem/action-regex-match@d50fd2e7a37d0e617aea3d7ada663bd56862b9cc
+        id: regex-match-cli-gh-issue
+        with:
+          text: ${{ github.event.pull_request.body }}
+          regex: "https:\/\/github.com\/forcedotcom\/cli\/issues\/[0-9]+"
```

```suggestion
          regex: "https:\/\/github\.com\/forcedotcom\/cli\/issues\/[0-9]+"
```

literal `.` https://regex101.com/r/RndU8S/1